src/project-management/project-management-internal.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '../firebase-app';
+import { FirebaseServiceInterface } from '../firebase-service';
+import { ProjectManagement } from './project-management';
+
+/**
+ * ProjectManagement service bound to the provided app.
+ */
+export class ProjectManagementService extends ProjectManagement implements FirebaseServiceInterface {
```

Comment:
Not sure if this is a good idea to have for consistency

Reply:
Let's not add these right now. It's easy enough to add them later.